packages/format-library/src/text-color/index.js

```diff
@@ -94,7 +119,7 @@ function TextColorEdit( {
 export const textColor = {
 	name,
 	title,
-	tagName: 'span',
+	tagName: 'mark',
```

Comment:
Is there any reason why this was changed from a `span`- to `mark`-tag? I cant find any information in #34680 to it. 

This looks like a big breaking change, also due the fact that some normalize css files might contain default styling like backgroundColor or color to those elements.

Reply:
@ellatrix